tests/simple/testdata/timestamps.textproto

```diff
@@ -138,3 +168,101 @@ section {
     value: { bool_value: false }
   }
 }
+section {
+  name: "timestamp_arithmetic"
+  description: "Arithmetic operations on timestamps and/or durations."
+  test {
+    name: "add_duration_to_time"
+    expr: "timestamp('2009-02-13T23:00:00Z')+duration('240s') == timestamp('2009-02-13T23:04:00Z')"
```

Comment:
Try moving the expected value into the "value:" field below, by looking up the structure of a Timestamp message.  This might not work due to a limitation in the proto library for Go, but try it anyhow.